base/compiler/utilities.jl

```diff
@@ -433,13 +433,16 @@ function find_ssavalue_uses(e::Expr, uses::Vector{BitSet}, line::Int)
         elseif isa(a, SSAValue)
             push!(uses[a.id], line)
         elseif isa(a, Expr)
-            find_ssavalue_uses(a, uses, line)
+            find_ssavalue_uses!(a, uses, line)
         end
     end
 end
 
-function find_ssavalue_uses(e::PhiNode, uses::Vector{BitSet}, line::Int)
-    for val in e.values
+function find_ssavalue_uses!(e::PhiNode, uses::Vector{BitSet}, line::Int)
+    values = e.values
+    for i = 1:length(values)
+        isassigned(values) || continue
+        val = values[i]
```

Comment:
Implemented a fix for the segmentation fault occurring in `test/compiler/ssair.jl`.